README.md

```diff
@@ -107,6 +112,7 @@ var bundler = new require('polymer-bundler')({
   excludes: [],
   inlineScripts: true,
   inlineCss: true,
+  rewriteUrlsInTemplates: false,
```

Comment:
Could you split this example up into smaller examples for each documented constructor/method? 

Added bonus: It will help separate the three different things being documented here